README.md

```diff
@@ -22,13 +22,12 @@ platforms):
      consult [the `python-oracledb` documentation][5] for information about the
      versions of Instant Client that work. Typically, an Instant Client a few
      versions later than what is officially supported should work fine.
+   - Modern versions of `node-oracledb` have a "thin mode" that does not
+     require the Instant Client to be installed, but as of the time of writing,
+     the version of Oracle Database used by MIT is too old to support it.
```

Comment:
As far as I could gather, the issue is not just that the version of Oracle Database is too old, but that it uses a proprietary encryption algorithm that is only supported in the thick mode.